### PR TITLE
HZX-287: Data policies demo

### PR DESCRIPTION
**Pull request by pmcgleenon**
https://hazelcast.atlassian.net/browse/HZX-287

Added the stock ticker demo to illustrate how Data Policies work.   Components include:
 - Flow
 - Management Center
 - kafka
 - stock-ticker producer

There is a taxi project with a definition for the Stock Ticker events and some example policies

### CODE REVIEW

File: stock-ticker/README.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# Hazelcast Flow Stock-Ticker Demo 
+
+This repo contains services required for the Hazelcast Flow stock-ticker demo.
+
+This demo includes an example Taxi project which reads events from a Kafka topic
+and shows how data policies can be used to control access to data in the stream
+based on the User's Role.
```
Comment (amandalindsay) [inline stock-ticker/README.md:7]:
```suggestion
based on the user's role.
```

File: stock-ticker/README.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+Flow is configured with an Identity Provider in this demo.  The Identity Provider
+is Hazelcast Management Center which is configured with these users:
+
+
+| User       | Role        | Description                      |
+|------------|-------------|----------------------------------|
+| admin      | Full access | Full access to all features      |
```
Comment (amandalindsay) [inline stock-ticker/README.md:15]:
```suggestion
| admin      | full access | Full access to all features      |
```

File: stock-ticker/README.md (hunk 1 above)
Comment (amandalindsay) [reply stock-ticker/README.md:7]: assuming we don't need to use initial caps on User's Role? Or is this a proper name?

File: stock-ticker/README.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+| FilterTicker         | Obfuscates the Ticker symbol when Admin role not present      |
+
+
+
+## Prerequisites
+
+To use these files, you must first have the following installed:
+
+- [Docker](https://docs.docker.com/engine/installation/)
+- [docker-compose](https://docs.docker.com/compose/install/)
```
Comment (amandalindsay) [inline stock-ticker/README.md:40]:
```suggestion
- [Docker-compose](https://docs.docker.com/compose/install/)
```

File: stock-ticker/README.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+## How to use
+
+The following steps will run a local instance of the Hazelcast Flow using the default configuration file (`docker-compose.yml`):
```
Comment (amandalindsay) [inline stock-ticker/README.md:44]:
```suggestion
The following steps will run a local instance of Hazelcast Flow using the default configuration file (`docker-compose.yml`):
```

File: stock-ticker/README.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+
+1. Clone this repository.
```
Comment (amandalindsay) [inline stock-ticker/README.md:46]:
```suggestion
1. Clone this repository
```

Comment (amandalindsay) [reply stock-ticker/README.md:46]: Remove full stops from end of bullets

**Review by amandalindsay — CHANGES_REQUESTED** [review]
looks good, just made some minor tweaks to capitalisation. Left 'Data Access Policies' as is, as assuming this is a proper name?

**Comment by pmcgleenon** [comment]
> looks good, just made some minor tweaks to capitalisation. Left 'Data Access Policies' as is, as assuming this is a proper name?

thanks for the feedback, I've applied those changes

**Review by amandalindsay — APPROVED** [review]

**Pull request merged**